src/test/java/calculator/NumbersTest.java

```diff
@@ -0,0 +1,30 @@
+package calculator;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class NumbersTest {
+
+    @Test
+    @DisplayName("sum 테스트")
+    void sum() {
+        List<Number> numberList = new ArrayList<>();
+        numberList.add(new Number(1));
+        numberList.add(new Number(2));
+        numberList.add(new Number(3));
```

Comment:
`Arrays.asList()` 를 사용해 보는 것은 어떨까요?

```suggestion
        List<Number> numberList = Arrays.asList(
                new Number(1),
                new Number(2),
                new Number(3)
        );
```